.github/workflows/test.yaml

```diff
@@ -126,4 +126,4 @@ jobs:
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
       # pr mode uses the HEAD of the branch, which is also correct for nightlies
```

Comment:
Is this comment still needed? It seems like it's wrong if we're using `main` for nightlies when it says to use `pr`.

Reply:
Removed it 👍